frontend/libs/studio-components/src/components/StudioTableRemotePagination/StudioTableRemotePagination.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { StudioTableRemotePagination } from './StudioTableRemotePagination';
+import { columns, rows } from './mockData';
+
+describe('StudioTableRemotePagination', () => {
+  const paginationProps = {
```

Comment:
Should we assign the type on this one as well? 😊

Reply:
Fixed 😊